### ISSUE TITLE
Installed new OCSP V5.3.7 using Binaries, but OCPS not working.

### ISSUE THREAD

**Issue by ohaya** [xipki/xipki#165]
I installed XiPKI 5.3.7 under tomcat 9.x, and completed the steps in the README.

However, when I test with "openssl ocsp", it is getting the following:

OCSP Request Data:
    Version: 1 (0x0)
    Requestor List:
        Certificate ID:
          Hash Algorithm: sha1
          Issuer Name Hash: 89A60AF2317750D6591B094B002D985B41AE50CD
          Issuer Key Hash: 64B19D8743AD5A6CF6F477D2960CF9A08A04B24C
          Serial Number: 17
    Request Extensions:
        OCSP Nonce:
            04105EA615200BBA350D5C873123CF55AA35
Responder Error: malformedrequest (1)

Any idea what would cause this problem?

Thanks,
Jim

**Comment by ohaya** [comment]
Also, it appears that when I touch the UPDATEME file in a crl-XXX directory, the OCSP is not picking up the CRL.

**Comment by ohaya** [comment]
Here's the ocsp.log from startup.  It doesn't look like it is recognizing/loading the CRL?  Also is that warning about the Mysql driver normal?

2020-05-27 09:48:21,571 INFO o.x.u.XipkiBaseDir [main] use basedir '/apps/apache-tomcat-9.0.20/xipki',
2020-05-27 09:48:21,649 INFO o.x.o.s.OcspServletFilter [main] logReqResp: false
2020-05-27 09:48:21,692 INFO o.x.s.Securities [main] add BouncyCastleProvider
2020-05-27 09:48:21,803 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs12.P12SignerFactory@56fe2233
2020-05-27 09:48:21,804 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.emulator.EmulatorP11ModuleFactory@28790ede
2020-05-27 09:48:21,805 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.iaik.IaikP11ModuleFactory@31d07dd9
2020-05-27 09:48:21,805 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.proxy.ProxyP11ModuleFactory@568c71de
2020-05-27 09:48:21,835 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs11.P11SignerFactory@3ab8f900
2020-05-27 09:48:21,855 INFO o.x.o.s.OcspServerImpl [main] starting OCSPResponder server ...
2020-05-27 09:48:22,937 INFO c.z.h.HikariDataSource [main] HikariPool-1 - Starting...
2020-05-27 09:48:22,944 WARN c.z.h.u.DriverDataSource [main] Registered driver with driverClassName=com.mysql.jdbc.Driver was not found, trying direct instantiation.
2020-05-27 09:48:23,108 INFO c.z.h.HikariDataSource [main] HikariPool-1 - Start completed.
2020-05-27 09:48:23,117 INFO o.x.o.s.s.CrlDbCertStatusStore [main] CertStore store1 not changed
2020-05-27 09:48:23,164 INFO o.x.o.s.s.DbCertStatusStore [main] Updated issuers of store store1
2020-05-27 09:48:23,165 INFO o.x.o.s.s.DbCertStatusStore [main] Updated CRL_INFOs of store store1
2020-05-27 09:48:23,167 INFO o.x.o.s.OcspServerImpl [main] started OCSPResponder server
2020-05-27 09:48:23,168 INFO o.x.o.s.OcspServletFilter [main] remote management is disabled

**Comment by ohaya** [comment]
I was able to get rid of the warning about the mysql driver by using com...cj... for the driver class, but still don't see it importing/registering the CRL:

2020-05-27 10:00:44,279 INFO o.x.u.XipkiBaseDir [main] use basedir '/apps/apache-tomcat-9.0.20/xipki',
2020-05-27 10:00:44,363 INFO o.x.o.s.OcspServletFilter [main] logReqResp: false
2020-05-27 10:00:44,410 INFO o.x.s.Securities [main] add BouncyCastleProvider
2020-05-27 10:00:44,528 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs12.P12SignerFactory@56fe2233
2020-05-27 10:00:44,529 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.emulator.EmulatorP11ModuleFactory@28790ede
2020-05-27 10:00:44,530 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.iaik.IaikP11ModuleFactory@31d07dd9
2020-05-27 10:00:44,530 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.proxy.ProxyP11ModuleFactory@568c71de
2020-05-27 10:00:44,569 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs11.P11SignerFactory@3ab8f900
2020-05-27 10:00:44,589 INFO o.x.o.s.OcspServerImpl [main] starting OCSPResponder server ...
2020-05-27 10:00:45,737 INFO c.z.h.HikariDataSource [main] HikariPool-1 - Starting...
2020-05-27 10:00:46,170 INFO c.z.h.HikariDataSource [main] HikariPool-1 - Start completed.
2020-05-27 10:00:46,179 INFO o.x.o.s.s.CrlDbCertStatusStore [main] CertStore store1 not changed
2020-05-27 10:00:46,229 INFO o.x.o.s.s.DbCertStatusStore [main] Updated issuers of store store1
2020-05-27 10:00:46,231 INFO o.x.o.s.s.DbCertStatusStore [main] Updated CRL_INFOs of store store1
2020-05-27 10:00:46,232 INFO o.x.o.s.OcspServerImpl [main] started OCSPResponder server
2020-05-27 10:00:46,234 INFO o.x.o.s.OcspServletFilter [main] remote management is disabled
2020-05-27 10:01:31,710 INFO o.x.u.XipkiBaseDir [main] use basedir '/apps/apache-tomcat-9.0.20/xipki',
2020-05-27 10:01:31,785 INFO o.x.o.s.OcspServletFilter [main] logReqResp: false
2020-05-27 10:01:31,789 INFO o.x.s.Securities [main] BouncyCastleProvider already added
2020-05-27 10:01:31,797 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs12.P12SignerFactory@1f682fee
2020-05-27 10:01:31,798 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.emulator.EmulatorP11ModuleFactory@94f30d
2020-05-27 10:01:31,798 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.iaik.IaikP11ModuleFactory@73177b3f
2020-05-27 10:01:31,798 INFO o.x.s.p.P11ModuleFactoryRegisterImpl [main] added P11ModuleFactory binding for org.xipki.security.pkcs11.proxy.ProxyP11ModuleFactory@18c43e27
2020-05-27 10:01:31,826 INFO o.x.s.SignerFactoryRegisterImpl [main] added SignerFactory binding for org.xipki.security.pkcs11.P11SignerFactory@de7b93a
2020-05-27 10:01:31,837 INFO o.x.o.s.OcspServerImpl [main] starting OCSPResponder server ...
2020-05-27 10:01:32,077 INFO c.z.h.HikariDataSource [main] HikariPool-2 - Starting...
2020-05-27 10:01:32,094 INFO c.z.h.HikariDataSource [main] HikariPool-2 - Start completed.
2020-05-27 10:01:32,102 INFO o.x.o.s.s.CrlDbCertStatusStore [main] CertStore store1 not changed
2020-05-27 10:01:32,117 INFO o.x.o.s.s.DbCertStatusStore [main] Updated issuers of store store1
2020-05-27 10:01:32,118 INFO o.x.o.s.s.DbCertStatusStore [main] Updated CRL_INFOs of store store1
2020-05-27 10:01:32,120 INFO o.x.o.s.OcspServerImpl [main] started OCSPResponder server
2020-05-27 10:01:32,121 INFO o.x.o.s.OcspServletFilter [main] remote management is disabled

Also, I checked the ocspcrl database in Mysql and all tables are empty still?

Jim
